modules/yieldmoBidAdapter.js

```diff
@@ -211,6 +214,53 @@ function createNewVideoBid(response, bidRequest) {
     ttl: TIME_TO_LIVE,
     vastXml: response.adm
   };
+
+  const playbackmethodMap = [
+    {
+      preload: true,
+      mute: false,
+      autoPlay: true
+    },
+    {
+      preload: true,
+      mute: true,
+      autoPlay: true
+    }
+  ];
+
+  if (imp.placement !== 1) {
```

Comment:
@ym-abaranov we should use the context parameter instead of placement for this logic, currently we only support "instream" but with this it will be "outstream"

Reply:
Nevermind, the above comment. Please ignore it.